docs/UserGuide.md

```diff
@@ -130,85 +138,94 @@ Format: `list`
 
 Edits an existing patient's details in the address book.
 
+ * Edits the person with the specified name or id.
+ * If an invalid name or id is passed, an error message will be logged.
+ * At least one of the optional fields must be provided.
+ * Existing fields will be updated to the input values.
+ * If the fields do not exist, the corresponding field with details will be added.
+
 Format: `edit n/NAME or id/IC_NUMBER [field] ...`
 
-* Edits the person with the specified name or id.
-* If an invalid name or id is passed, an error message will be logged.
-* At least one of the optional fields must be provided.
-* Existing fields will be updated to the input values.
-* If the fields do not exist, the corresponding field with details will be added.
+ * See `add` for the list of fields that can be edited.
 
 Examples:
-*  `edit n/John Doe phone/91234567 email/johndoe@example.com` Edits the phone number and email address of patient with name `John Doe` to be `91234567` and `johndoe@example.com` respectively.
-*  `edit id/S8943782H  ward/Psychology ` Edits the ward of the patient with id `S8943782` to be `Psychology`.
+ * `edit n/John Doe phone/91234567 email/johndoe@example.com`
+   * Edits the phone number and email address of patient with name `John Doe` to be `91234567` and
+     `johndoe@example.com` respectively.
+
+ * `edit id/S8943782H  ward/Psychology `
+   * Edits the ward of the patient with id `S8943782` to be `Psychology`.
 
 ### Locating persons by name: `find`
 
-Enables front desk workers to quickly retrieve patient information based on command executed. If no fields are provided, the function returns all the patient’s related information. Else, only the details relating to the field are provided.
+Enables front desk workers to quickly retrieve patient information based on command executed.
+If no fields are provided, the function returns all the patient’s related information.
+Else, only the details relating to the field are provided.
 
 Patient information includes:
-- Patient’s contacts
-- Patient’s medical history
-- Patient’s ward information
-- Patient’s upcoming appointment time(s)
+ - Patient’s contacts
+ - Patient’s medical history
+ - Patient’s ward information
+ - Patient’s upcoming appointment time(s)
```

Comment:
These lines appear to be duplicated in lines 203-206. Consider consolidating these lines of code around line 60 to provide a more comprehensive explanation of all the possible fields. This will help clarify the purpose of each field, making it easier to understand their usage:

| Field         | Description                              |
|------------|----------------------------------|
| e/              | Email Address                           |
| p/              | Phone Number                        |
| a/             | Home Address                          |
| m/            | Medical history                        |
| w/             | Ward information                    |
| start/from/ | Start and end appointment times |
| t/               | Tag                                           |

Besides, I do not think we have the fields table in the UG yet also.